docs/styles/transitions.md

```diff
@@ -72,6 +76,68 @@ Instead of this
 }
 ```
 
+#### Styling the overlay and content
+
+It is possible to independently style the transitions for the modal 
+overlay and modal content. The example below will fade the overlay 
+in and slide the modal content on-screen from the bottom of the page:
+
+```javascript
+{
+  <Modal
+    style={{
+      overlay: { background: 'rgba(0, 0, 0, 0.3)' },
+      content: {
+        top: '50%',
+        left: '50%',
+        right: 'auto',
+        bottom: 'auto',
+        marginRight: '-50%',
+        // transform: 'translate(-50%, -50%)', This will override the css values if not removed
```

Comment:
Maybe we can move this note after the first paragraph to warn about the precedence.